gate-web/src/main/groovy/com/netflix/spinnaker/gate/security/iap/IAPSsoConfig.java

```diff
@@ -62,6 +64,19 @@
   @Autowired
   IAPSecurityConfigProperties configProperties;
 
+  @Bean
+  public IAPAuthenticationFilter iapAuthenticationFilter() {
+    return new IAPAuthenticationFilter(
+      configProperties, permissionService, front50Service);
+  }
+
+  @Bean
+  public FilterRegistrationBean registration(IAPAuthenticationFilter filter) {
+    FilterRegistrationBean registration = new FilterRegistrationBean(filter);
+    registration.setEnabled(false);
```

Comment:
Why not enabled?

Reply:
because otherwise it would add an extra filter in the WebSecurity filter chain